SingularityService/src/main/java/com/hubspot/singularity/config/SingularityConfiguration.java

```diff
@@ -2084,11 +2085,19 @@ public void setUseCaffeineCache(boolean useCaffeineCache) {
     this.useCaffeineCache = useCaffeineCache;
   }
 
-  public int getCaffeineCacheTtl() {
-    return caffeineCacheTtl;
+  public int getDeployCaffeineCacheTtl() {
+    return deployCaffeineCacheTtl;
   }
 
-  public void setCaffeineCacheTtl(int caffeineCacheTtl) {
-    this.caffeineCacheTtl = caffeineCacheTtl;
+  public void setDeployCaffeineCacheTtl(int deployCaffeineCacheTtl) {
+    this.deployCaffeineCacheTtl = deployCaffeineCacheTtl;
+  }
+
+  public int getRequestCaffeineCacheTtl() {
+    return requestCaffeineCacheTtl;
```

Comment:
One last nit — can we append the time unit to this so users don't have to search code for it?